app/alembic/versions/20241026151704_e824fba2b86a_new_migration.py

```diff
@@ -0,0 +1,30 @@
+"""New migration
+
+Revision ID: 20241026151704_e824fba2b86a
```

Comment:
is this file still needed? just undo this migration and generate a new one instead of fixing this in the next one